static/js/components/Embedly_test.js

```diff
@@ -35,7 +35,10 @@ describe("Embedly", () => {
     const wrapper = renderEmbedly(image)
     assert.ok(wrapper.find(".photo"))
     const img = wrapper.find("img")
-    assert.equal(img.props().src, image.url)
+    assert.ok(img.props().src.includes(encodeURIComponent(image.url)))
+    assert.ok(
+      img.props().src.startsWith("https://i.embed.ly/1/display/resize/")
+    )
```

Comment:
I might suggest, for this test, stubbing out the `embedlyImage` function and just checking that `img.props().src` is equal to whatever the return value of the stubbed function is (otherwise this test here is kind of indirectly also testing the implementation of the `embedlyImage` function)

Reply:
Stubbed out